neon_tts_plugin_coqui_ai/__init__.py

```diff
@@ -70,30 +70,50 @@ def __init__(self, lang="en", config=None):
         self.cache_engines = config.get("cache", True)
 
     def get_tts(self, sentence: str, output_file: str, speaker: Optional[dict] = None):
-        stopwatch = Stopwatch()
-        speaker = speaker or dict()
-
         # TODO: speaker params are optionally defined and should be handled whenever defined
         # # Read utterance data from passed configuration
         # request_lang = speaker.get("language",  self.lang)
         # request_gender = speaker.get("gender", "female")
         # request_voice = speaker.get("voice")
 
-        synthesizer, tts_kwargs = self._init_model(speaker)
-
         to_speak = format_speak_tags(sentence)
         LOG.debug(to_speak)
         if to_speak:
-            with stopwatch:
-                wav_data = synthesizer.tts(sentence, **tts_kwargs)
-            LOG.debug(f"TTS Synthesis time={stopwatch.time}")
+            wav_data, synthesizer = self.get_audio(sentence, speaker, audio_format = "internal")
 
-            with stopwatch:
-                synthesizer.save_wav(wav_data, output_file)
-            LOG.debug(f"File access time={stopwatch.time}")
+            self._audio_to_file(wav_data, synthesizer, output_file)
 
         return output_file, None
 
+    def get_audio(self, sentence: str, speaker: Optional[dict] = None, audio_format: str = "internal"):
```

Comment:
Please add a docstring here. Also, consider a boolean param like `as_file` or `as_data` instead of `audio_format` since this is a binary setting (also means you don't have to add an `else` case to handle unexpected values)

Reply:
Later there will be third `bytes` type 
Just not sure yeat how to better cover it

Reply:
Added docstring https://github.com/NeonGeckoCom/neon-tts-plugin-coqui-ai/pull/6/commits/91217577006b885caca42efdfbf506dae987b83c